fastlane/lib/util.rb

```diff
@@ -11,9 +12,7 @@ def authorize_ci_for_keys
 
 # Gets the version, either from Travis or from Hockey
 def current_build_number(**args)
-  if ENV.key?('TRAVIS_BUILD_NUMBER')
-    return ENV['TRAVIS_BUILD_NUMBER']
-  end
+  return ENV['TRAVIS_BUILD_NUMBER'] if ENV.key?('TRAVIS_BUILD_NUMBER')
```

Comment:
I get what this does, but I think the expanded form is more understandable to non-ruby devs, which is what most of us are. 

And it's not a big deal to go either way.